File: INICIANTE/JAVASCRIPT/1017.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var tempo = prompt("tempo");
var velocidade = prompt("VELOCIDADE");
var distancia = tempo * velocidade;
var litros = distancia / 12;

console.log(litros.toFixed(3));

File: INICIANTE/JAVA/1017.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int tempo, velocidade;
        double distancia, litros;
        Scanner sc = new Scanner(System.in);
        tempo = sc.nextInt();
        velocidade = sc.nextInt();
        distancia = tempo*velocidade;
        litros = distancia / 12;
        System.out.println(String.format("%.3f", litros));
    }
}
